pycobertura/reporters.py

```diff
@@ -246,6 +283,13 @@ def get_report_lines(self):
 
         return lines
 
+    def per_file_stats(self, lines):
+        items = lines.items()
+        number_of_files = len(self.differ.files()) - 1
+        rows = [{k: v[i] for k, v in items} for i in range(number_of_files)]
+        footer = {k: v[number_of_files] for k, v in items if k != "Missing"}
+        return {"files": rows, "total": footer}
```

Comment:
Can we apply the same variable naming convention as you used in the other function? The changes you did above look great.

Reply:
updated both functions a little and used same naming convention. Please re-review!